mono/utils/networking-posix.c

```diff
@@ -95,9 +97,11 @@ mono_get_address_info (const char *hostname, int port, int flags, MonoAddressInf
 		if (cur->family == PF_INET) {
 			cur->address_len = sizeof (struct in_addr);
 			cur->address.v4 = ((struct sockaddr_in*)res->ai_addr)->sin_addr;
+#ifdef HAVE_STRUCT_SOCKADDR_IN6			
```

Comment:
Hard to believe there are platforms without ipv6 but ok.

Reply:
Yes, we support at least two of them. :)